src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -681,4 +682,15 @@ private String generateCoOwnersEmailsLine(String courseId) {
         return coOwnersEmailsLine.substring(0, coOwnersEmailsLine.length() - 2);
     }
 
+    /**
+     * Generates additional contact information for User Email Templates.
+     * @param course Details in {@code course}.
+     * @return A String containing the contact information after replacing the placeholders.
+     */
+    private String fillUpAdditionalContactInformationFragment(CourseAttributes course, String emailBody) {
```

Comment:
Do you really need the second parameter if it is always `EmailTemplates.FRAGMENT_SESSION_ADDITIONAL_CONTACT_INFORMATION`?

Reply:
Oh yes. I overlooked that. I will fix this. :sweat_smile: